.gitignore

```diff
@@ -41,3 +48,13 @@ object_database/*
 
 *.pyc
 *.pyo
+
+cscope.out
+tags
+
+# auto-generated files during building
+CMakeDoxyfile.in
+libraries/egenesis/egenesis_brief.cpp
+libraries/egenesis/egenesis_full.cpp
+libraries/egenesis/embed_genesis
+tests/generate_empty_blocks/generate_empty_blocks
```

Comment:
Changing .gitignore is unrelated, please undo.

Reply:
I think the changes are good. However, better to do it with another PR.